src/ds/stacktrace_utils.h

```diff
@@ -31,20 +31,4 @@ namespace logger
     print_stacktrace();
     exit(1);
   }
-
-  static inline void Init(const char* file_name_extention)
-  {
-    signal(SIGSEGV, handler); // install our handler
```

Comment:
Should `handler()` go too?

Reply:
Sure, unless this is still used for debugging. @ashamis?

Reply:
This should be replaced by 

#ifndef INSIDE_ENCLAVE
"backward::SignalHandling sh;"
#endif

from https://github.com/bombela/backward-cpp